ViewModel/Adminhtml/Merchandising/Page.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Algolia\AlgoliaSearch\ViewModel\Adminhtml\Merchandising;
+
+use Algolia\AlgoliaSearch\Helper\ProxyHelper;
+
+class Page
+{
+    /** @var ProxyHelper */
+    private $proxyHelper;
+
+    public function __construct(
+        ProxyHelper $proxyHelper
+    ) {
+        $this->proxyHelper = $proxyHelper;
+    }
+
+    /**
+     * @return bool
+     */
+    public function canAccessLandingPageBuilder()
+    {
+        $planLevel = 1;
+
+        $planLevelInfo = $this->proxyHelper->getInfo(ProxyHelper::INFO_TYPE_PLAN_LEVEL);
```

Comment:
`planLevel` is included in `ALL` and can be get from `ProxyHelper:: getClientConfigurationData()` as well. Could it be refactored and do only one request to Proxy? :)

Reply:
Oh amazing. Can I update @damcou 's  LPB to do the same? Or do you want that in a separate PR?

Reply:
Feel free to update it here :)